fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1010,6 +1010,10 @@ alter_table_clause ::=
     {:
         RESULT = new EnableFeatureClause(featureName, properties);
     :}
+    | KW_MODIFY KW_DISTRIBUTION opt_distribution:distribution
```

Comment:
Please modify the document of Alter stmt in English and Chinese.

Reply:
DONE!